security/cmd/node_agent_k8s/main.go

```diff
@@ -85,6 +88,10 @@ func init() {
 
 	rootCmd.PersistentFlags().StringVar(&serverOptions.CAProviderName, "caProvider", caProvider, "CA provider")
 	rootCmd.PersistentFlags().StringVar(&serverOptions.CAEndpoint, "caEndpoint", caAddr, "CA endpoint")
+	rootCmd.PersistentFlags().StringArrayVar(&serverOptions.PluginNames, "pluginNames",
+		[]string{""}, "authN provider specific plugin names")
```

Comment:
Considering using authentication instead of authN in the comment to make it more readable.

Reply:
renamed.